account.cmd/src/main/java/com/ultimatesoftware/banking/account/cmd/domain/aggregates/Account.java

```diff
@@ -113,6 +118,64 @@ public void on(UpdateAccountCommand updateAccountCommand) {
         apply(new AccountUpdatedEvent(id, updateAccountCommand.getCustomerId()));
     }
 
+    @CommandHandler
+    public void on(StartTransferTransactionCommand command) throws AccountNotEligibleForDebitException {
+        if (!AccountRules.eligibleForDebit(this, command.getAmount())) {
+            apply(new TransferFailedToStartEvent(command.getTransactionId().toString()));
+            throw new AccountNotEligibleForDebitException(id, balance);
+        }
+
+        logger.info("Transfer transaction started from {} successfully", id);
+        apply(new TransferTransactionStartedEvent(command.getId(), command.getDestinationId(), command.getAmount(),
+                command.getTransactionId().toString()));
+    }
+
+    @CommandHandler
+    public void on(StartTransferDepositCommand startTransferDepositCommand) {
+        logger.info("Transfer to {} successfully", id);
+        double newBalance = balance - startTransferDepositCommand.getAmount();
+        apply(new TransferWithdrawConcludedEvent(startTransferDepositCommand.getId(), newBalance,
+                startTransferDepositCommand.getTransactionId().toString()));
+    }
+
+    @CommandHandler
+    public void on(ConcludeTransferCommand concludeTransferCommand) {
+        logger.info("Transfer concluded to {} successfully", id);
+        double newBalance = balance + concludeTransferCommand.getAmount();
+        apply(new TransferDepositConcludedEvent(concludeTransferCommand.getId(), newBalance,
```

Comment:
The command is missing information that the event has in the name, I fixed this one in a later commit.